### PR TITLE
added code of conduct

### PR DESCRIPTION
**Pull request by geek-2002**

### CODE REVIEW

**Comment by geek-2002** [comment]
please do let me know if there are any changes

**Comment by erictleung** [comment]
Thanks for the pull request. It is great that you took a look at the link and copied the entire code of conduct. But I think it would be simpler to just add the link and point to the live code of conduct page, similar to the main repository has done it here https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CODE_OF_CONDUCT.md.

All that should be necessary is this text in the `CODE_OF_CONDUCT.md` file

```
> Our Code of Conduct is available here: <https://www.freecodecamp.org/code-of-conduct/>
```

Let me know if you have any questions or comments about that.

**Comment by geek-2002** [comment]
okay

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# freeCodeCamp is a friendly place to learn to code. We’re committed to keeping it that way.
+
+# Contributor The freeCodeCamp Code of conduct
+ ## By using freeCodeCamp, you agree that you'll follow this code of conduct.
+
+ ## In short: Be nice. No harassment, trolling, or spamming.
+
+> Our Code of Conduct is available here: <https://www.freecodecamp.org/code-of-conduct/>
+
```
Comment (geek-2002) [inline CODE_OF_CONDUCT.md:9]: I have simply added the link to code of conduct of freeCodeCamp

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+available at https://www.freecodecamp.org/news/code-of-conduct/
+
```
Comment (geek-2002) [inline CODE_OF_CONDUCT.md:14]: Please let me know if the attribution part is required or not

**Pull request merged**